languages/ruby/lib/oso/oso.rb

```diff
@@ -204,6 +204,36 @@ def authorized_query(actor, action, resource_cls) # rubocop:disable Metrics/Meth
         .build_query
     end
 
+    def authzd_query(actor, action, resource_cls) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
Before merging, we need to change this to a better user facing name. Same for `authzd_resources`.

Not sure what's good let's just address this last or maybe as another PR b/c it's going to be a big diff. But should do a quick follow up before we do the user facing interface (in case that's not done by end of cycle).